compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/TypeChecker.java

```diff
@@ -227,6 +233,14 @@ public TypeChecker(CompilerContext context) {
         this.typeParamAnalyzer = TypeParamAnalyzer.getInstance(context);
         this.anonymousModelHelper = BLangAnonymousModelHelper.getInstance(context);
         this.semanticAnalyzer = SemanticAnalyzer.getInstance(context);
+        initModifierFunctions();
+    }
+
+    private void initModifierFunctions() {
```

Comment:
We could initialize this in a static block.

Reply:
fixed by d9d1222ebd2f15e8eb8cc80333d36825d92b488a